### ISSUE TITLE
Wurst Setup doesn't show error pulling dependecy with only a main branch

### ISSUE THREAD

**Issue by Jaccouille** [wurstscript/WurstSetup#67]
Wurst Setup GUI pulls content from `master` branch by default, but if a repository does not have a `master` branch, it doesn't show any error in the logs :

![22-03-13-19-44-35](https://user-images.githubusercontent.com/7768858/158076560-3c13c299-0bd5-4ff7-b164-8dcbf5b7706b.jpg)
_Red rectangle highlight empty dependencies folders_

This can be confusing especially as new github repositories got their principal branch named `main` by default instead of `master`.

**Comment by Frotty** [comment]
> This can be confusing especially as new github repositories got their principal branch named main by default instead of master.

Yes, you can thank github for this entire unnecessary fuckery from their side breaking tooling and whatnot just to avoid the word `master`. But it can be configured in your user settings.

As shown in the readme you must append the branch name like so:
![image](https://user-images.githubusercontent.com/1486037/158078743-4419ca19-1155-4b92-9b7c-a6b84927bd43.png)

Back then there was no easy way to get the "default" branch with jgit, dunno if that changed.
